### PR TITLE
Change Chrome to Firefox in a couple of comments within Firefox classes 

### PR DESCRIPTION
**Pull request by jakedurand**
- [X ] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

Changed Chrome to Firefox in a couple of comments within Firefox classes.

### CODE REVIEW

File: dotnet/src/webdriver/Firefox/FirefoxOptions.cs [hunk 1]
```diff
@@ -255,8 +255,8 @@ public void SetPreference(string preferenceName, string preferenceValue)
         /// geckodriver.exe.</remarks>
         public override void AddAdditionalCapability(string capabilityName, object capabilityValue)
         {
-            // Add the capability to the chromeOptions object by default. This is to handle
-            // the 80% case where the chromedriver team adds a new option in chromedriver.exe
+            // Add the capability to the FirefoxOptions object by default. This is to handle
+            // the 80% case where the Firefoxdriver team adds a new option in Firefoxdriver.exe
```
Comment (cgoldberg) [inline dotnet/src/webdriver/Firefox/FirefoxOptions.cs]: shouldn't this be geckodriver?

Comment (jakedurand) [reply dotnet/src/webdriver/Firefox/FirefoxOptions.cs]: Good point. Fixed.

**Comment by jimevans** [comment]
@jakedurand Would it be possible to get you to squash these down to a single commit? Thanks.

**Comment by jimevans** [comment]
Merged in c15cdbc. Thank you for your contribution.

**Pull request closed without merging**